WebService/appsettings.ini

```diff
@@ -12,6 +12,25 @@ iothub_data_folder = ./data/iothub/
 twin_read_write_enabled = "${?PCS_TWIN_READ_WRITE_ENABLED}"
 
 
+[DeviceSimulationService:Deployment]
+# AAD Domain of the Azure subscription where the Azure IoT Hub is deployed.
+# The value is optional because the service can be deployed without a hub.
+# The info is used to create a URL taking to the IoT Hub metrics in the Azure portal.
+azure_subscription_domain = "${?PCS_SUBSCRIPTION_DOMAIN}"
+# Azure subscription where the Azure IoT Hub is deployed, e.g. "mytest.onmicrosoft.com".
+# The value is optional because the service can be deployed without a hub.
+# The value is used to create a URL taking to the IoT Hub metrics in the Azure portal.
+azure_subscription_id = "${?PCS_SUBSCRIPTION_ID}"
+# Azure resource group where the Azure IoT Hub is deployed, e.g. "abcd1234-5678-1234-abcd-abcd5678abcd".
+# The value is optional because the service can be deployed without a hub.
+# The value is used to create a URL taking to the IoT Hub metrics in the Azure portal.
+azure_resource_group = "${?PCS_RESOURCE_GROUP}"
+# IoT Hub name, e.g. "mytest3507e89".
+# The value is optional because the service can be deployed without a hub.
+# The value is used to create a URL taking to the IoT Hub metrics in the Azure portal.
+azure_iothub_name = "${?PCS_IOHUB_NAME}"
```

Comment:
Is this value required? The hub name can be extracted from the connection string.

Reply:
I think it might be different from the one used in the connection string, e.g. in cases where the name is long